docs/guide/tutorial-1-your-first-simulation-in-webots-20-minutes.md

```diff
@@ -1,211 +1,172 @@
 ## Tutorial 1: Your First Simulation in Webots (20 Minutes)
 
-In this first tutorial, you will create your first simulation.
-This simulation will contain a simple environment (a light and an arena with floor and walls), a predefined robot (e-puck) and a controller program that will make the robot move (see [this figure](#what-you-should-see-at-the-end-of-the-tutorial)).
-The objective of this tutorial is to familiarize yourself with the user interface and with the basic concepts of Webots.
-
-%figure "What you should see at the end of the tutorial."
+The objective of this first tutorial is to familiarize yourself with the user interface and with the basic concepts of Webots.
+You will create your first simulation containing a simple environment: an arena with floor and walls, a few boxes, an e-puck robot and a controller program that will make the robot move.
 
+%figure "What you should see at the end of the first tutorial."
 ![tutorial_e-puck.png](images/tutorial_e-puck.png)
-
 %end
 
-### Create a New World
-
-A World contains information like where the objects are, what they look like, how they interact with each other, what is the sky color, where is the gravity vector, etc.
-
-> **Theory**: A **World** defines the initial state of a simulation.
-The different objects are called **Nodes** and are organized hierarchically in a **Scene Tree**.
-It means that a node can have some sub-nodes.
-
-<!-- -->
-
-> **Note**: A world is stored in a file having the ".wbt" extension.
-The file format is derived from the **VRML97** language, and is human readable.
-The world files must be stored directly in the project subdirectory called "worlds".
-
-Webots is currently open and runs an arbitrary simulation.
-
-> **Hands on**: Pause the current simulation by clicking on the `Pause` button of the 3D view.
-The simulation is paused if the virtual time counter on the main toolbar is stable.
-
-<!-- -->
-
-> **Hands on**: Create a new world by selecting the `File / New World` menu item.
-
-A new world is now open.
-For now, the 3D window displays nothing.
-The Scene Tree view (on the left hand side) currently lists the fundamental nodes:
-
-- [WorldInfo](../reference/worldinfo.md): containing misc global parameters.
-- [Viewpoint](../reference/viewpoint.md): defining the main camera parameters.
+### Start Webots
 
-As no light and no 3D object are defined, the entire scene is empty, and thus nothing is displayed.
+If not already done, download and install Webots, following these [installation instructions](installing-webots.md).
 
-Each node has some customizable properties called **fields**.
-The first step is about modifying the background color.
+> **Hands on #1**: Start Webots by double-clicking on its icon (or invoking it from a command line in a Terminal).
```

Comment:
We should link to https://cyberbotics.com/doc/guide/starting-webots?version=fix-tutorial-1